src/config.hpp.in

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+/** This file is automatically processed by CMake.
+ *  Make any changes to src/config.hpp.in, not config.hpp
+ */
+
+#include <filesystem>
+
+#cmakedefine INSTALL_RESOURCES
+
+static const std::filesystem::path resourceDir("${RESOURCE_DIR}");
```

Comment:
If possible `constexpr auto const`, if not could you make them `auto const`?

Reply:
std::filesystem::path isn't suitable for constexpr, made them auto const.